src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -84,7 +84,7 @@ public class XdocsPagesTest {
 
     private static final Path AVAILABLE_CHECKS_PATH = Paths.get("src/xdocs/checks.xml");
     private static final String LINK_TEMPLATE =
-            "(?s).*<a href=\"config_\\w+\\.html#%1$s\">(\\s)*%1$s</a>.*";
+            "(?s).*<a href=\"[^\"]+#%1$s\">([\\n\\s])*%1$s([\\n\\s])*</a>.*";
```

Comment:
Most `<a` lines are now longer and need to be separated into multiple lines to not violate 100 char rule
```
            <td>
              <a href="checks/annotation/annotationusestyle.html#AnnotationUseStyle">
                AnnotationUseStyle
              </a>
            </td>
```

Also, we remove `config_` prefix.

Reply:
This fixes test 
```
[ERROR]   XdocsPagesTest.testAllChecksPresentOnAvailableChecksPage:232->lambda$testAllChecksPresentOnAvailableChecksPage$1:236 AnnotationUseStyle is not correctly listed on Available Checks page - add it to src/xdocs/checks.xml
```